policies/image/fbc_labels.repo

```diff
@@ -0,0 +1,10 @@
+package fbc_checks
+
+violation_maintainer_required[{"msg": msg, "details":{"name": name, "description": description, "url": url}}] {
+  not input.Labels["operators.operatorframework.io.index.configs.v1"]
+
+  name := "operators_operatorframework_io_index_configs_v1_label_required"
+  msg := "The 'operators.operatorframework.io.index.configs.v1' label should be defined for FBC image"
+  description := "Should set DC-specific label `operators.operatorframework.io.index.configs.v1` for the location of the DC root directory for FBC image."
+  url := "https://source.redhat.com/groups/public/container-build-system/container_build_system_wiki/guide_to_layered_image_build_service_osbs#jive_content_id_Labels"
```

Comment:
Maybe we can use the `https://docs.openshift.com/container-platform/4.9/operators/admin/olm-managing-custom-catalogs.html#olm-creating-fb-catalog-image_olm-managing-custom-catalogs` as the url here

Reply:
Thank you! Let me update it.

Reply:
Thanks!